activities/MindMath.activity/js/components/SugarPresence.js

```diff
@@ -61,7 +61,6 @@ Vue.component('sugar-presence', {
 				network.createSharedActivity(vm.bundleId, function (groupId) {
 					console.log("Activity shared");
 					vm.isHost = true;
-					vm.$emit('multiplayer-game-started');
```

Comment:
@dhruvmisra ,add this line. After adding this line, the game is working perfectly.

Reply:
Umm, it's probably better not to modify these components as they will be updated quite often. For this use case specifically, I added the `journal-shared-instance` event from SugarJournal. It can be handled similar to `journal-new-instance` handler. We can call the `onMultiplayerGameStarted()` inside that handler.